pkg/aws/kms.go

```diff
@@ -0,0 +1,114 @@
+package aws
+
+import (
+	"context"
+	"crypto/aes"
+	"crypto/cipher"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/kms"
+	"github.com/aws/aws-sdk-go-v2/service/kms/types"
+)
+
+// KMSClient maintains the client to access AWS KMS related service.
+type KMSClient struct {
+	client *kms.Client
+	// keyAlias the alias name of the KMS master key.
+	keyAlias string
+}
+
+// NewKMSClient construct a KMS client.
+func NewKMSClient(ctx context.Context, keyAlias, region string) (*KMSClient, error) {
+	cfg, err := config.LoadDefaultConfig(ctx, config.WithRegion(region))
```

Comment:
How about factoring the code in https://github.com/llmariner/common/blob/main/pkg/aws/s3.go#L30-L55 and use that to construct the config?